docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc

```diff
@@ -88,7 +88,7 @@ You can attach tags to the application manually, instead of enabling automated t
 +
 [NOTE]
 ====
-Analysis engines use standard rules for a comprehensive set of migration targets, but if the target is not included or is a customized framework, custom rules can be added. Custom rules files are validated.
+Analysis engines use standard rules for a comprehensive set of migration targets, but if the target is not included or is a customized framework, custom rules can be added. Only manually uploaded custom rule files are validated.
```

Comment:
Hi @emarcusRH, a file can have more than one rule, so "custom rules files"

Reply:
I understand, but it is incorrect grammar - in English , the word "files" is enough to indicate that you have multiple instances of a custom rule.

Reply:
I see. Good to know!
Thanks